karyon3-core/src/main/java/com/netflix/karyon/ArchaiusKaryonConfiguration.java

```diff
@@ -0,0 +1,72 @@
+package com.netflix.karyon;
+
+import java.util.List;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Module;
+import com.google.inject.name.Names;
+import com.netflix.archaius.guice.ArchaiusModule;
+
+public class ArchaiusKaryonConfiguration extends DefaultKaryonConfiguration {
+    private static final String DEFAULT_CONFIG_NAME = "application";
```

Comment:
There's 3 places where "application" is defined (2 here and 1 in DefaultKaryonConfiguration). Perhaps changes these to reference a single def?